src/commands/createVirtualMachine/VirtualMachineCreateStep.ts

```diff
@@ -50,13 +52,27 @@ export class VirtualMachineCreateStep extends AzureWizardExecuteStep<IVirtualMac
 
         const rgName: string = nonNullValueAndProp(context.resourceGroup, 'name');
 
-        const creatingVm: string = localize('creatingVm', `Creating new virtual machine "${vmName}"...`);
-        const createdVm: string = localize('creatingVm', `Created new virtual machine "${vmName}".`);
+        const creatingVm: string = localize('creatingVm', 'Creating new virtual machine "{0}"...', vmName);
+        const creatingVmDetails: string = localize(
+            'creatingVmDetails',
+            'Creating new virtual "{0}" with size "{1}" and image "{2}"',
+            vmName,
+            nonNullProp(hardwareProfile, 'vmSize').replace(/_/g, ' '), // sizes are written with underscores as spaces
+            `${nonNullProp(storageProfile, 'imageReference').offer} ${nonNullProp(storageProfile, 'imageReference').sku}`);
+
+        const createdVm: string = localize('creatingVm', 'Created new virtual machine "{0}".', vmName);
 
-        ext.outputChannel.appendLog(creatingVm);
+        ext.outputChannel.appendLog(creatingVmDetails);
         progress.report({ message: creatingVm });
         context.virtualMachine = await computeClient.virtualMachines.createOrUpdate(rgName, vmName, virtualMachineProps);
         ext.outputChannel.appendLog(createdVm);
+
+        // Note: intentionally not waiting for the result of this before returning
+        window.showInformationMessage(createdVm, viewOutput).then(async (result: MessageItem | undefined) => {
+            if (result === viewOutput) {
```

Comment:
These should be wrapped in `callWithTelemetryAndErrorHandling` like here: https://github.com/microsoft/vscode-azureappservice/blob/master/src/commands/createWebApp/showCreatedWebAppMessage.ts#L21

Otherwise error message won't be shown (not likely for showing output, but still possible in the future)